src/sinks/elasticsearch/encoder.rs

```diff
@@ -317,4 +348,33 @@ mod tests {
         assert!(nested.contains_key("_type"));
         assert_eq!(nested.get("_type").unwrap().as_str(), Some("TYPE"));
     }
+
+    #[test]
+    fn encodes_fields_with_newlines() {
+        let mut writer = Vec::new();
+
+        _ = write_bulk_action(
+            &mut writer,
+            "ACTION\n",
+            "INDEX\n",
+            "TYPE\n",
+            false,
+            &DocumentMetadata::Id("ID\n".to_string()),
+        );
```

Comment:
Should we add an `.unwrap()` here?

Reply:
Mmm, yeah, I just used the pattern the other tests here are using, but yes, it would make sense to use `unwrap` on all of these